resources/web/docs_js/components/console_widget.jsx

```diff
@@ -33,23 +33,25 @@ export class _ConsoleForm extends Component {
 
     return (
       <form>
+         <p><strong>Want to use the Kibana Console?</strong></p>
         <label for="url">{props.langStrings(props.url_label)}</label>
         <input
           id="url"
           type="text"
           value={getValueFromState('url')}
           onInput={linkState(this, getFieldName('url'))}
         />
-
-        <label for="curl_host">curl {props.langStrings('host')}</label>
+        <p></p>
+        <p><strong>Want to use curl? (basic auth)</strong></p>
+        <label for="curl_host">Elasticsearch {props.langStrings('host')}</label>
         <input
           id="curl_host"
           type="text"
           value={getValueFromState('curl_host')}
           onInput={linkState(this, getFieldName('curl_host'))}
         />
 
-        <label for="curl_username">curl {props.langStrings('username')}</label>
+        <label for="curl_username">Elasticsearch {props.langStrings('username')}</label>
         <input
           id="curl_username"
           type="text"
```

Comment:
Below `Elasticsearch username` field:

For settings information, go to [Configure Kibana](https://www.elastic.co/guide/en/kibana/master/settings.html).

**Save** | **Reset**

Reply:
Is it confusing to send users to the Kibana docs to learn more about Elasticsearch settings? Yes. Is there a better place to send them? I'm not sure.

Reply:
Maybe we can trust the users that if they have Elasticsearch and Kibana running, they'll be able to find this information. If they don't have a running deployment yet, we've got links for them to get started. At that point, they're in another domain (setting up a deployment) rather than the domain of configuring the example widget settings.